docs/getting-started.md

```diff
@@ -191,6 +192,41 @@ these components have their own Makefiles. In those cases you can place a
 `.custom_image` file in the directory with the base image you wish to use and
 that will be picked up by the Makefile and build the container.
 
+#### Building binaries and images for non linux/amd64 architecture
+
+*\*Useful for Apple Silicon chips users.*
+
+###### Containers
+
+If you need your images to be built for non linux/amd64 architecture,
+you can supply the flag `CONTAINER_OS_ARCH` for customisation of containers.
+
+This can be passed to the make commands used to build images, for example:
+
+```bash
+make CONTAINER_OS_ARCH=linux/arm64 components
```

Comment:
Optional suggestion: maybe it will be clearer for users if we give an example of exactly how to set it up for your mac. Currently the example just shows the default value, but it might be clearer to say that it might work for mac this way.